### PR TITLE
Feature/driver

### PR DESCRIPTION
**Pull request by VaradharajanThiagarajan**
implemented post and get method for the driver. please merge this branch to the main. Also included the RESTDOCS

### CODE REVIEW

**Comment by raghav-codes** [comment]
Approved.

**Review by raghav-codes — DISMISSED** [review]

**Review by 404-bug-not-found — DISMISSED** [review]
LGTM

File: build.gradle [hunk 1]
```diff
@@ -1,4 +1,4 @@
-plugins {
+;plugins {
```
Comment (davidgrieser) [inline build.gradle]: @raghav-codes / @404-bug-not-found did you notice this semi-colon? It clearly doesn't fail the build, but seems odd.

**Review by davidgrieser — DISMISSED** [review]
Nice clean PR

**Review by davidgrieser — APPROVED** [review]
Looks Good To Merge!

**Review by 404-bug-not-found — APPROVED** [review]
LGTM

**Pull request merged**